src/main/java/pages/HomePage.java

```diff
@@ -1,6 +1,32 @@
 package pages;
 
 import base.BasePage;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
 
 public class HomePage extends BasePage {
+    public HomePage homePage;
+
+    final String accounts = "Accounts";
+    @FindBy(xpath = "//div[@class='bBottom']//a[@title='Home']")
+    public WebElement homeNavBar;
+
+    @FindBy(xpath = "//div[@class='slds-icon-waffle']")
+    public WebElement iconAppLouncher;
+
+    @FindBy(xpath = "//input[@type='search'][@class='slds-input']")
```

Comment:
Use AND instead of ][

Reply:
@Evangard  Can't find element when adding AND